### DISCUSSION TITLE
Links into other repositories become a dead end

### DISCUSSION THREAD

**Discussion by cmahnke** [community/community#56547] [Mobile]
### Select Topic Area

Bug

### Body

I'm not sure if this a real bug since I assume that this is expected behavior, that haven't thought thru to the end:
When a repository links to another one, then the link back at the top refers to the previous repository, it's not possible to navigate the linked repository.
One Example is in https://github.com/IIIF/iiif-3d-stories/blob/master/README.md which is displayed when opening  https://github.com/IIIF/iiif-3d-stories in the mobile app. On the top of this document is the current location, since the original repository is deprecated. When one follows the link on the mobile app, there is no way to get other contents of the linked repository (https://github.com/IIIF/3d/issues), there is only a link back.

That's not what a user expects...

**Comment by HenSquared** [comment]
This is a great feedback.I am able to reproduce this on iOS. I'm not sure exactly the best behavior for this situation is. I'm going to create an issue and discuss internally with the team. Thanks!
